cilium-dbg/cmd/bpf_multicast_subscribers.go

```diff
@@ -0,0 +1,294 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Authors of Cilium
+
+package cmd
+
+import (
+	"errors"
+	"fmt"
+	"io/fs"
+	"net/netip"
+	"os"
+	"sort"
+	"text/tabwriter"
+
+	"github.com/spf13/cobra"
+	"github.com/vishvananda/netlink"
+
+	"github.com/cilium/cilium/pkg/command"
+	"github.com/cilium/cilium/pkg/common"
+	"github.com/cilium/cilium/pkg/defaults"
+	maps_multicast "github.com/cilium/cilium/pkg/maps/multicast"
+)
+
+const (
+	allGroupsKW = "all"
+	remoteKW    = "Remote Node"
+	localKW     = "Local Endpoint"
+)
+
+var BpfMcastSubscriberCmd = &cobra.Command{
+	Use:     "subscriber",
+	Aliases: []string{"sub"},
+	Short:   "Manage the multicast subscribers.",
+}
+
+func init() {
+	BpfMcastSubscriberCmd.AddCommand(MulticastGroupSubscriberListCmd)
+	command.AddOutputOption(MulticastGroupSubscriberListCmd)
+	BpfMcastSubscriberCmd.AddCommand(MulticastGroupSubscriberAddCmd)
+	BpfMcastSubscriberCmd.AddCommand(MulticastGroupSubscriberDelCmd)
+}
+
+var MulticastGroupSubscriberListCmd = &cobra.Command{
+	Use:     "list < group | all >",
+	Aliases: []string{"ls"},
+	Short:   "List the multicast subscribers for the given group.",
+	Long: `List the multicast subscribers for the given group. 
+To get the subscribers for all the groups, use 'cilium-dbg bpf multicast subscriber list all'.`,
+	Run: func(cmd *cobra.Command, args []string) {
+		common.RequireRootPrivilege("cilium bpf multicast subscriber list")
+
+		groupAddr, all, err := parseMulticastGroupSubscriberListArgs(args)
+		if err != nil {
+			Fatalf("invalid argument: %s\n", err)
+		}
+
+		groupV4Map, err := maps_multicast.OpenGroupV4OuterMap(maps_multicast.GroupOuter4MapName)
+		if err != nil {
+			if errors.Is(err, fs.ErrNotExist) {
+				fmt.Fprintln(os.Stderr, "Cannot find multicast bpf maps")
+				return
+			}
+			Fatalf("Failed to open multicast bpf maps: %s", err)
+		}
+
+		var groups []netip.Addr
+		if all {
+			groups, err = groupV4Map.List()
+			if err != nil {
+				Fatalf("Failed to list multicast groups: %s", err)
+			}
+		} else {
+			groups = append(groups, groupAddr)
+		}
+
+		err = outputGroups(groupV4Map, groups)
+		if err != nil {
+			Fatalf("Failed to list multicast subscribers: %s", err)
+		}
+	},
+}
+
+var MulticastGroupSubscriberAddCmd = &cobra.Command{
+	Use:   "add <group> <subscriber-address>",
+	Short: "Add a remote subscriber to the given multicast group.",
+	Long: `Add a remote subscriber to the given multicast group.
+Only remote subscribers are added via command line. Local subscribers will be automatically populated in the map based on IGMP messages.
+Remote subscriber is typically other Cilium nodes. To add remote subscriber, following information is required:
+- group: multicast group address to which subscriber is added.
+- subscriber-address: cilium internal IP of remote node.
+
+For Example: 
+To add a remote node 10.100.0.1 to multicast group 229.0.0.1, use the following command:
+	cilium-dbg bpf multicast subscriber add 229.0.0.1 10.100.0.1
```

Comment:
```suggestion
Example: `To add a remote node 10.100.0.1 to multicast group 229.0.0.1, use the following command:

	cilium-dbg bpf multicast subscriber add 229.0.0.1 10.100.0.1`
```
Can you make it an Example field, for consistency with add/delete group commands?